accountsplus/forms.py

```diff
@@ -1,9 +1,20 @@
 from __future__ import unicode_literals
 
 import django.forms
+from django.conf import settings
+from django.apps import apps
+
 from captcha.fields import ReCaptchaField
 
 
 class CaptchaForm(django.forms.Form):
     captcha = ReCaptchaField()
     username = django.forms.CharField()
+
+    def clean_username(self):
+        username = self.cleaned_data.get('username')
+        User = apps.get_model(getattr(settings, 'AUTH_USER_MODEL'))
+        exists = User.objects.filter(email=username).exists()
```

Comment:
you don't need to store exists in a local variable. you can just use it in the if test below.